src/triggers.js

```diff
@@ -373,48 +373,54 @@ function userIdForLog(auth) {
   return auth && auth.user ? auth.user.id : undefined;
 }
 
-function logTriggerAfterHook(triggerType, className, input, auth) {
+function logTriggerAfterHook(triggerType, className, input, auth, logLevel) {
   const cleanInput = logger.truncateLogMessage(JSON.stringify(input));
-  logger.info(
-    `${triggerType} triggered for ${className} for user ${userIdForLog(
-      auth
-    )}:\n  Input: ${cleanInput}`,
-    {
-      className,
-      triggerType,
-      user: userIdForLog(auth),
-    }
-  );
+  if (logLevel !== 'none') {
+    logger[logLevel](
+      `${triggerType} triggered for ${className} for user ${userIdForLog(
+        auth
+      )}:\n  Input: ${cleanInput}`,
+      {
+        className,
+        triggerType,
+        user: userIdForLog(auth),
+      }
+    );
+  }
 }
 
-function logTriggerSuccessBeforeHook(triggerType, className, input, result, auth) {
+function logTriggerSuccessBeforeHook(triggerType, className, input, result, auth, logLevel) {
   const cleanInput = logger.truncateLogMessage(JSON.stringify(input));
   const cleanResult = logger.truncateLogMessage(JSON.stringify(result));
-  logger.info(
-    `${triggerType} triggered for ${className} for user ${userIdForLog(
-      auth
-    )}:\n  Input: ${cleanInput}\n  Result: ${cleanResult}`,
-    {
-      className,
-      triggerType,
-      user: userIdForLog(auth),
-    }
-  );
+  if (logLevel !== 'none') {
+    logger[logLevel](
```

Comment:
Would it be better to pass the log level `none` also to the logger and handle it there? This way we don't need the `if` statement everywhere we log something.

Reply:
I think you're right, it would be better to multualize but I'm not sur it's the right way : It seems weird to me to add a ```none()``` function that do nothing.
Maybe I was just wrong in my analysis and the 'none' is useless : if you don't want to save a log, you just have to switch the level of the trigger to 'silly' or 'debug'. What do you think ?

Reply:
Just commit a fix in this way, I think it's more consistent.

Reply:
Some log level definitions have a concept of `none` or `off`. From the logger's perspective it seems odd, because the logger should be called when there's something to log. From the developer's perspective it makes more sense, because they want to control whether something is logged at all or if yes at which log level.

We can solve this in 2 ways:
- a) Never call the logger directly anywhere in code; instead add a proxy method that which includes the `if "none", don't log" condition.
- b) Just add this condition to the logger itself, to allow passing the developer's choice directly to the logger.

Option (a) seems to include a lot of code changes, and we have no easy way to enforce not calling the logger directly when reviewing PRs. So I'd go with option (b).